src/status_im/ui/screens/profile/events.cljs

```diff
@@ -29,10 +29,11 @@
 (handlers/register-handler-fx
   :profile/send-transaction
   [trim-v]
-  (fn [_ [chat-id]]
-    {:dispatch [:navigate-to :chat chat-id]
-     ;;TODO get rid of timeout
-     :dispatch-later [{:ms 100 :dispatch [:select-chat-input-command {:name "send"}]}]}))
+  (fn [{:keys [db]} [chat-id]]
+    (let [send-command (first (get-in db [:contacts/contacts "transactor-personal" :commands :send]))]
```

Comment:
Not related to PR, though is there a reason why `(get-in db [:contacts/contacts "transactor-personal" :commands :send])` is a collection?

Reply:
The reason is that with command scopes, the bot/dapp could potentially define multiple commands with the same name (but different scope).
I think that putting them all into collection is wrong and we should key them by the scope object, but that will be the part of my next refactoring, it will also simplify the mechanism of selecting relevant commands quite a lot.

Reply:
Agree with this point (although I think that there will be a lot of problems with this), but disagree that it's something that should be refactored now.

Reply:
Don't worry @alwx it won't be any big refactoring, just simple change that will make looking-up commands by #{name scope} set easier.